cmd/build/helpers.go

```diff
@@ -0,0 +1,8 @@
+package main
+
+import "path/filepath"
+
+func dotFilepathJoin(elem ...string) string {
```

Comment:
method only used for packages, not file paths, would say that there is no correct use for it